### DISCUSSION TITLE
Limit number of updates per MR when grouping dependencies

### DISCUSSION THREAD

**Discussion by CCMackenzie** [renovatebot/renovate#23220] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have configured Renovate to group my dependencies by manager in order to reduce merge request (MR) noise. However, I occasionally encounter merge requests with a large number of changes (around 10), which can result in build errors that are challenging to diagnose due to the number of simultaneous updates. Is there a way to configure a maximum group size, so that dependencies are still grouped together in merge requests, but limited to batches of 5 updates per merge request?

### What you tried so far.

I have not found anything so far that would make this possible, the only limits I have seen are prHourly and prConcurrent.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
no
